pkg/controllers/manifestwork.go

```diff
@@ -131,7 +141,10 @@ func (r *HypershiftDeploymentReconciler) createMainfestwork(ctx context.Context,
 }
 
 func (r *HypershiftDeploymentReconciler) deleteManifestworkWaitCleanUp(ctx context.Context, hyd *hypdeployment.HypershiftDeployment) (ctrl.Result, error) {
-	m := ScafoldManifestwork(hyd)
+	m, err := ScafoldManifestwork(hyd)
```

Comment:
Scaffold

Reply:
Done, I thought I fixed all of them, T.T